packages/web-components/fast-components-msft/src/accordion/accordion-item/accordion-item.styles.ts

```diff
@@ -4,14 +4,14 @@ import {
     focusVisible,
     forcedColorsStylesheetBehavior,
 } from "@microsoft/fast-foundation";
+import { SystemColors } from "@microsoft/fast-web-utilities";
```

Comment:
This change needed?

Reply:
wait? maybe I didn't accidentally do this, I definitely didn't mean to, will remove